src/clj/datahike_server/core.clj

```diff
@@ -2,18 +2,10 @@
   (:gen-class)
   (:require [mount.core :as mount]
             [taoensso.timbre :as log]
-            [datahike-server.config :refer [config]]
-            [datahike-server.database]
```

Comment:
both, `datahike-server.database` and `datahike-server.server` needs to be in here so the components can be mounted as you already mentioned in the tests.